src/FNA3D_Driver_Vulkan.c

```diff
@@ -2060,6 +2069,20 @@ static uint8_t VULKAN_INTERNAL_QuerySwapChainSupport(
 	SwapChainSupportDetails *outputDetails
 ) {
 	VkResult result;
+	VkBool32 supportsPresent;
+
+	renderer->vkGetPhysicalDeviceSurfaceSupportKHR(
+		physicalDevice,
+		renderer->queueFamilyIndex,
+		surface,
+		&supportsPresent
+	);
+
+	if (!supportsPresent)
+	{
+		FNA3D_LogError("This surface does not support presenting!");
```

Comment:
This can warn, will error later
```suggestion
		FNA3D_LogWarn("This surface does not support presenting!");
```